client/src/executor.js

```diff
@@ -4,7 +4,7 @@ var vm = require('vm');
 
 module.exports = {
   template: function(code, template){
-    var context = template.type === 'jade' ? { jade: require('jade/runtime.js')} : {};
+    var context = template.type === 'jade' ? { jade: require('./renderers/support/jade-runtime.js')} : {};
```

Comment:
True, this is the only thing I need from Jade. Including it as whole dependency means including a series of sub-dependencies I don't care about which contains some known security vulnerabilities - which is the reason we shrinkwrapped the deps in the first place. This is going to be cleaned-up as part of the Pug migration.